src/sidebar/components/annotation-body.js

```diff
@@ -50,7 +50,6 @@ export default function AnnotationBody({
             textClass={{
               'annotation-body__text': true,
               'is-hidden': isHidden(annotation),
-              'has-content': text.length > 0,
```

Comment:
This component is only rendered if the annotation has content, so this stateful class is meaningless.